### PR TITLE
UIRouter and NotifyRouter mappings in extended status

### PR DESCRIPTION
**Pull request by pzduniak**

### CODE REVIEW

**Comment by pzduniak** [comment]
There were no prior tests, not sure if we need any for this. Manually tested out, works OK.

**Comment by mmaxim** [comment]
Going to change to @joshblum here since he changes this stuff a lot.

File: go/client/cmd_status.go [hunk 1]
```diff
@@ -220,6 +221,7 @@ func (c *CmdStatus) load() (*fstatus, error) {
 	status.LocalDbStats = extStatus.LocalDbStats
 	status.LocalChatDbStats = extStatus.LocalChatDbStats
 	status.CacheDirSizeInfo = extStatus.CacheDirSizeInfo
+	status.UIRouterMapping = extStatus.UiRouterMapping
```
Comment (joshblum) [inline go/client/cmd_status.go]: you should log this value as well as the new info in `ClientStatus`, the `status` command should have an output that is similar to the result in the log send

Comment (pzduniak) [reply go/client/cmd_status.go]: Will do, I wasn't sure about it and assumed I'd get an answer in the review :)

**Comment by pzduniak** [comment]
Sample output:

```
desktop:
    /mnt/c/gopath/src/github.com/keybase/client/shared/node_modules/electron/dist/electron --type=renderer --no-sandbox --enable-features=SharedArrayBuffer --service-pipe-token=14494516367994659720 --lang=en-US --app-path=/mnt/c/gopath/src/github.com/keybase/client/shared/node_modules/electron/dist/resources/default_app.asar --node-integration=true --webview-tag=true --no-sandbox --background-color=#fff --num-raster-threads=4
--enable-main-frame-before-activation --service-request-channel-token=14494516367994659720 --renderer-client-id=6 --shared-files=v8_context_snapshot_data:100,v8_natives_data:101 [cid: 1, pid: 1779, version: Development, description: Main Renderer, uis: RekeyUI,HomeUI,ChatUI,SecretUI,IdentifyUI, notifications: Session,Users,Kbfs,Kbfsdesktop,Tracking,Service,App,Chat,PGP,Badges,Reachability,Team,Chatattachments,Wallet]
```

**Comment by pzduniak** [comment]
Tests probably should get fixed when it gets merged, I think?

**Comment by joshblum** [comment]
@pzduniak rebase/merge master in and tests should pass

File: go/client/cmd_status.go [hunk 2]
```diff
@@ -345,8 +355,36 @@ func (c *CmdStatus) outputClients(dui libkb.DumbOutputUI, clients []keybase1.Cli
 		if len(cli.Details.Desc) > 0 {
 			dstr = ", description: " + cli.Details.Desc
 		}
-		dui.Printf("    %s [pid: %d%s%s]\n", strings.Join(cli.Details.Argv, " "), cli.Details.Pid, vstr, dstr)
+		if uis, ok := mappedUIs[cli.ConnectionID]; ok {
+			dstr += ", uis: " + strings.Join(uis, ",")
```
Comment (joshblum) [inline go/client/cmd_status.go]: small nit, put a space after the comma. can this and then notifications be on a separate line?

File: go/client/cmd_status.go [hunk 3]
```diff
@@ -324,27 +327,64 @@ func (c *CmdStatus) outputTerminal(status *fstatus) error {
...
+
+	items := []string{}
+	for i := 0; i < value.NumField(); i++ {
+		if casted, ok := value.Field(i).Interface().(bool); ok && casted {
+			items = append(items, typ.Field(i).Name)
 		}
-		dui.Printf("    %s [pid: %d%s%s]\n", strings.Join(cli.Argv, " "), cli.Pid, vstr, dstr)
 	}
+
+	return strings.Join(items, ",")
```
Comment (joshblum) [inline go/client/cmd_status.go]:
```suggestion
	return strings.Join(items, ", ")
```

File: go/client/cmd_status.go (hunk 2 above)
Comment (pzduniak) [reply go/client/cmd_status.go]: Sure, I removed the space to distinguish the separators between list items and items in the client desc. Separate lines look better indeed!

**Pull request merged**